hax/treemakers/posrec.py

```diff
@@ -0,0 +1,200 @@
+import hax
+import numpy as np
+from hax.minitrees import TreeMaker
+from pax.PatternFitter import PatternFitter
+from pax.configuration import load_configuration
+from pax.InterpolatingMap import InterpolatingMap
+from pax import utils
+from pax import exceptions
+from pax.plugins.interaction_processing.S1AreaFractionTopProbability import binom_test as binom_test_pax
+from keras.models import model_from_json
+import tensorflow as tf
+
+class PositionReconstruction(TreeMaker):
+    """Stores position-reconstruction-related variables.
+
+    Provides:
+       - s1_pattern_fit_hax: S1 pattern likelihood computed with corrected
+                             position and areas
+       - s1_pattern_fit_hits_hax: S1 pattern likelihood computed with corrected
+                                  position and hits
+       - s1_pattern_fit_bottom_hax: S1 pattern likelihood computed with corrected
+                                    position and bottom array area
+       - s1_pattern_fit_bottom_hits_hax: S1 pattern likelihood computed with corrected
+                                         position and bottom array hits
+
+       - s1_area_fraction_top_probability_hax: S1 AFT computed with corrected position
+
+       - x_observed_nn_tf: TensorFlow NN reconstructed x position
+       - y_observed_nn_tf: TensorFlow NN reconstructed y position
+       
+       - s1_area_upper_injection_fraction: s1 area fraction near Rn220 injection points (near PMT 131)
+       - s1_area_lower_injection_fraction: s1 area fraction near Rn220 injection points (near PMT 243)
+    """
+    __version__ = '0.15'
+    extra_branches = ['peaks.area_per_channel[260]',
+                      'peaks.hits_per_channel[260]',
+                      'peaks.n_saturated_per_channel[260]',
+                      'interactions.x', 'interactions.y', 'interactions.z']
+
+    def __init__(self):
+
+        hax.minitrees.TreeMaker.__init__(self)
+        
+        # We need to pull some stuff from the pax config
+        self.pax_config = load_configuration("XENON1T")
+        self.tpc_channels = list(range(0, 247 + 1))
```

Comment:
You can fetch this from pax config in previous line

Reply:
Fixed in https://github.com/XENON1T/hax/pull/174/commits/d61afe00aa7246043a4fd1b58e7bbf1cef93ce48